test.go

```diff
@@ -133,6 +133,17 @@ func (client *Client) ReadTest(test *Test) (*Test, error) {
 	return readTest, nil
 }
 
+// ReadTests reads all tests for a bucket
```

Comment:
Please can you add the reference to the api documentation, similar to how the other public methods have

i.e. 

`// ReadTests reads all tests for a bucket. See: https://www.runscope.com/docs/api/tests#list